code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -751,8 +758,15 @@ var/list/robot_verbs_default = list(
 	var/mob/living/M = user
 	if(!opened)//Cover is closed
 		if(locked)
-			to_chat(user, "You emag the cover lock.")
-			locked = 0
+			if(ert_upgrade)
+				if(prob(20))

```

Comment:
Yuck. 20% chance of the emag working to unlock the cover? If you successfully disable the ERT borg you shouldn't be countered by some RND nonsense 👎

Reply:
Right now, a skilled player can emag a borg so fast they barely have a chance to react. 

ERT borgs are stronger than regular borgs.

They also don't get the normal safeguards that normal borgs get. Normal borgs, you can notice they are emagged if you open them. People (except ERT members) can't open ERT borgs. Normal borgs, you can notice something is off if they aren't synched to the AI. ERT borgs are never synched to the AI.

There is literally no way of knowing that an ERT borg is emagged.
When it is emagged, it is even stronger than a regular emagged borg.
And it has a lower chance of being discovered.

For all of these reasons... it makes total sense that they would be harder to emag than a regular borg.

Reply:
Harder perhaps and I wouldn't disagree with that. What I don't agree with is the implementation of a percentage RNG check for it which completely disregards the skill of the player. 

>Right now, a skilled player can emag a borg in seconds.

The keywords here are "a skilled player". In my opinion we should reward skilled play, not place arbitrary uncontrollable RNG barriers to stumble them. This change will already give ERT borgs an incredible buff by unlocking their full capability. Making them harder to take down, and thus should be a greater reward for the antagonist who manages to do so. If the borg is alone and gets disabled then that's the fault of the ERT for not working as a team. There's no way that someone could take down and emag an ERT borg working in tandem with the other members of ERT.

Reply:
Do you have a suggestion for a mechanic that would make them harder to emag, but be skill-based?

Reply:
Maybe get the user to have to solve a simple puzzle or something when trying to open the cover. Similar to how the crates for mining work perhaps? I'm not sure how you would do it fast enough so that emagging the borg is still reliable within the stun-period of EMP/Flash etc.

Reply:
How about they have to unlock the cover in order to emag it, but they are require Central Level (ERT) access to open it. Meaning they would have to steal from or kill an ERT member. That can be the skill we are looking for.

Reply:
Bird, mini-games are not within scope. Jovaniph, ERT borgs already require ERT access to open, but emag gets around that. I can make it not get around that.... but that pretty much requires you to take out an ERT member for their ID, which some might argue is more difficult than this.